datalake/staging_stack.py

```diff
@@ -23,3 +27,11 @@ def __init__(self, scope: core.Construct, stack_id: str, **kwargs: Any) -> None:
             removal_policy=core.RemovalPolicy.DESTROY,
         )
         Tags.of(self.staging_bucket).add("ApplicationLayer", "storage")  # type: ignore[arg-type]
+
+        aws_ssm.StringParameter(
+            self,
+            "staging-bucket-arn",
+            description=f"Staging Bucket ARN for {deploy_env}",
+            parameter_name=STAGING_BUCKET_PARAMETER,
+            string_value=self.staging_bucket.bucket_arn,
+        )
```

Comment:
As discussed, it looks like this is only used elsewhere in CDK, not at runtime. We should avoid this overhead and complexity.